packages/store-engine-dexie/src/index.test.ts

```diff
@@ -61,7 +61,7 @@ describe('IndexedDBEngine', () => {
   describe('init', () => {
     it('resolves with the database instance to which the records will be saved.', async () => {
       engine = new IndexedDBEngine();
-      const instance = await engine.init(STORE_NAME);
+      const instance: Dexie = await engine.init(STORE_NAME);
```

Comment:
Do we need to cast to `Dexie` or can we work with the generic?

Reply:
`init()` has no generic type and it returns `any`. So to have types in the tests, I added this cast.